### PR TITLE
 in cosmosdb add relationpship publicipaddress

### PR DESCRIPTION
**Pull request by sagarnavale**

### CODE REVIEW

**Review by ganeshjagdale9545 — COMMENTED** [review]
code looks good can you test before merging on your local machine

File: cartography/intel/azure/cosmosdb.py [hunk 1]
```diff
@@ -152,6 +152,29 @@ def sync_database_account_data_resources(
         _load_database_account_write_locations(neo4j_session, database_account, azure_update_tag)
         _load_database_account_read_locations(neo4j_session, database_account, azure_update_tag)
         _load_database_account_associated_locations(neo4j_session, database_account, azure_update_tag)
+        _load_dadabase_account_associated_iprules(neo4j_session, database_account, azure_update_tag)
+
+
+@timeit
+def _load_dadabase_account_associated_iprules(neo4j_session: neo4j.Session, database_account: Dict, azure_update_tag: int,
```
Comment (mpurusottamc) [inline cartography/intel/azure/cosmosdb.py:159]: spelling mistake. fix this please.

File: cartography/intel/azure/cosmosdb.py [hunk 2]
```diff
@@ -152,6 +152,29 @@ def sync_database_account_data_resources(
         _load_database_account_write_locations(neo4j_session, database_account, azure_update_tag)
         _load_database_account_read_locations(neo4j_session, database_account, azure_update_tag)
         _load_database_account_associated_locations(neo4j_session, database_account, azure_update_tag)
+        _load_dadabase_account_associated_iprules(neo4j_session, database_account, azure_update_tag)
```
Comment (mpurusottamc) [inline cartography/intel/azure/cosmosdb.py:155]: spelling mistake here as well.

**Review by mpurusottamc — CHANGES_REQUESTED** [review]
minor fixes needed

**Review by ganeshjagdale9545 — APPROVED** [review]
Looks good

**Pull request merged**